### ISSUE TITLE
Extra field to store the contact person for an specific project

### ISSUE THREAD

**Issue by benjagm** [cncf/landscape2#618]
Hi Team,

Could it be possible to store the contact person for a certain landscape project? We found the mailing_list_url field but is not enough for us. We are wondering if there is a way to present Name and email from the person championing the adoption in the organizacion.

Maybe we can add it in the summary_use_case field with a markdown or html link but we are unsure if this link will be rendered considering that links are not rendered when added in the description field. Does the landscape support any kind of style rendering like markdown or something else?

Thanks a lot!

**Comment by tegioz** [comment]
Hi @benjagm 👋

Let us think a bit about this and we'll get back to you 🙂

**Comment by tegioz** [comment]
We've added a new field named `other_links` to the `extra` section that allows defining additional links for an item. You can use it to add the adoption champion if you'd like (or any other helpful links for your use case) 🙂

```yaml
extra:
  other_links:
    - name: "Adoption champion"
      url: "mailto:..."
``` 

<img width="2083" alt="link" src="https://github.com/cncf/landscape2/assets/1213902/2164d1e6-2299-4872-850d-d27d88fcbc49">

Closed by #624
